### PR TITLE
🌐 [i18n-DE] Translate docs to German

### PR DESCRIPTION
**Pull request by martinbrose**
References #1645 

**Key features**

- Translates main `README` to German;
- Translates the main landing docs (`index`, `installation`, `quick-start`) to German;
- Translates the How-To-Guides (`./guides`) to German

### CODE REVIEW

**Comment by Wauplin** [comment]
Gently pinging @patrickvonplaten @philschmid if you have time for a German-docs review :hugs:

**Review by patrickvonplaten — APPROVED** [review]
Sehr gut!

**Comment by BenjaminBossan** [comment]
Total unwichtig, aber ich finde "der [Hugging Face] Hub" klingt merkwürdig, ich hätte "das Hub" verwendet. "Der Hub" klingt so wie die Substantivierung von "Heben", also wie singular "Hub", plural "Hübe".

Ich hätte auch eher die Du-Form gewählt als die Sie-Form.

**Review by Wauplin — COMMENTED** [review]
Thanks for the contribution @martinbrose :hugs: 

I only reviewed the technical aspect of the PR which overall looks good (see comments). One more thing to do is to update the github config so that the DE docs get built. You need to update [this workflow](https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/upload_pr_documentation.yaml) and [this one](https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/build_documentation.yaml#L16) to add "de" as a language (as done in [`transformers`](https://github.com/huggingface/transformers/blob/main/.github/workflows/build_documentation.yml#L18)). Thanks in advance!

**Comment by martinbrose** [comment]
> Total unwichtig, aber ich finde "der [Hugging Face] Hub" klingt merkwürdig, ich hätte "das Hub" verwendet. "Der Hub" klingt so wie die Substantivierung von "Heben", also wie singular "Hub", plural "Hübe".
> 
> Ich hätte auch eher die Du-Form gewählt als die Sie-Form.

Hi @BenjaminBossan,

danke für Dein Feedback!

Ich hatte auch erst bei dem richtigen Artikel für Hub gestutzt.
Aber denke "der Hub" stimmt mit dem Duden überein, welcher dies für zwei Varianten so benutzt:
1. [der Hub als der zentrale Umschlagplatz / der Knotenpunkt](https://www.duden.de/rechtschreibung/Hub_Knotenpunkt)
2. [der Hub als der Digital Hub](https://www.duden.de/rechtschreibung/Digital_Hub)

Bei dem "Du" vs "Sie" habe ich mich an der deutschen Übersetzung der Transformers Bibliothek orientiert: https://huggingface.co/docs/transformers/v4.33.0/de/quicktour
Aber ich ändere es auch gerne zu "Du" falls wir zu dem Konsens kommen.

Viele Grüsse

**Comment by BenjaminBossan** [comment]
> Bei dem "Du" vs "Sie" habe ich mich an der deutschen Übersetzung der Transformers Bibliothek orientiert: https://huggingface.co/docs/transformers/v4.33.0/de/quicktour
> Aber ich ändere es auch gerne zu "Du" falls wir zu dem Konsens kommen.

So maybe a more general question to the others: Do we prefer the more formal German "Sie" or the informal German "du"? It's like "vous" vs "tu" in French. The transformers docs use "Sie"  and "vous" but IMHO it sounds very old school and incongruent with the general tone we use.

> Ich hatte auch erst bei dem richtigen Artikel für Hub gestutzt.
> Aber denke "der Hub" stimmt mit dem Duden überein, welcher dies für zwei Varianten so benutzt:

TIL!

File: README_de.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+</p>
+
+<p align="center">
+    <a href="https://huggingface.co/docs/huggingface_hub/index"><img alt="Documentation" src="https://img.shields.io/website/http/huggingface.co/docs/huggingface_hub/index.svg?down_color=red&down_message=offline&up_message=online&label=doc"></a>
+    <a href="https://github.com/huggingface/huggingface_hub/releases"><img alt="GitHub release" src="https://img.shields.io/github/release/huggingface/huggingface_hub.svg"></a>
+    <a href="https://github.com/huggingface/huggingface_hub"><img alt="PyPi version" src="https://img.shields.io/pypi/pyversions/huggingface_hub.svg"></a>
+    <a href="https://pypi.org/project/huggingface-hub"><img alt="downloads" src="https://static.pepy.tech/badge/huggingface_hub/month"></a>
+    <a href="https://codecov.io/gh/huggingface/huggingface_hub"><img alt="Code coverage" src="https://codecov.io/gh/huggingface/huggingface_hub/branch/main/graph/badge.svg?token=RXP95LE2XL"></a>
+</p>
+
```
[the remark this answers is not in the document]
Comment (martinbrose) [reply README_de.md:18]: Looking at this README.md files again, do we also need to change the links in the section "Key features" / "Hauptmerkmale"?
I guess the English version stays as is (as it's default?), but the German version probably needs a /de somewhere, once it's live?

[the remark this answers is not in the document]
Comment (Wauplin) [reply README_de.md:18]: Ah yes, good catch! You can already update all the `/docs` urls,  in `README_de.md` to redirect directly to the /de pages. That would be a good QOL improvement for German-speaking users. The urls don't exist yet but everything will settle once it's merged :)

[the remark this answers is not in the document]
Comment (martinbrose) [reply README_de.md:18]: Ok, think I've performed the right change.

And added /en for the English references in `README.md`,
and added /de for the German references in `README_de.md`.

**Comment by Wauplin** [comment]
Thanks for making the changes @martinbrose ! PR looks almost good to merge for me. Last thing to do, could you update the github workflow files to add "de" as a supported language as describe in https://github.com/huggingface/huggingface_hub/pull/1646#pullrequestreview-1622086609? Thanks in advance!

**Comment by martinbrose** [comment]
> Thanks for making the changes @martinbrose ! PR looks almost good to merge for me. Last thing to do, could you update the github workflow files to add "de" as a supported language as describe in [#1646 (review)](https://github.com/huggingface/huggingface_hub/pull/1646#pullrequestreview-1622086609)? Thanks in advance!

Done.

**Comment by martinbrose** [comment]
> > So maybe a more general question to the others: Do we prefer the more formal German "Sie" or the informal German "du"? It's like "vous" vs "tu" in French. The transformers docs use "Sie" and "vous" but IMHO it sounds very old school and incongruent with the general tone we use.
> 
> So I went to read French docs in `transformers` (and some Google docs) to get an idea. It feels formal but I'm fine with it TBH. It has more or less the same tone as reading a recipe so I was not shocked by it. I'd say both are fine for me, with a slight preference for keeping the same tone as the existing HF 🇩🇪 docs (so "Sie"?). In any case, it's not that much of a big deal IMO. @martinbrose you've done the work, I trust you to chose whichever you feel the most comfortable with.

Ok, I leave it at "Sie" for now. It's an easy change to "Du" if this is decided further along.

**Comment by BenjaminBossan** [comment]
> Ok, I leave it at "Sie" for now. It's an easy change to "Du" if this is decided further along.

This is a good job for an LLM ;-)

**Review by Wauplin — APPROVED** [review]
Final review done! Thanks for the effort @martinbrose, it's time to merge and publish that :tada: :tada: :tada:

**Pull request merged**